android-core/src/main/java/com/mparticle/identity/MParticleUserDelegate.java

```diff
@@ -303,4 +299,11 @@ public long getLastSeenTime(Long mpid) {
             return mConfigManager.getUserStorage(mpid).getLastSeenTime();
         }
     }
+
+    public AudienceTask<AudienceResponse> getUserAudiences(long mpId) {
+        if (mMessageManager != null && mMessageManager.mUploadHandler != null) {
+          return   mMessageManager.mUploadHandler.fetchUserAudiences(mpId);
+        }
+        return null;
```

Comment:
should you complete the task immediate here with an error, rather than returning null..?